### ISSUE TITLE
NVDA stops reading when Show markup balloons options changed in Word 2016

### ISSUE THREAD

**Issue by Qchristensen** [nvaccess/nvda#6156]
When you press alt+r (Revision) then tm (Show Markup) then B (Balloon), there are three options "Show revisions in balloons", "Show all revisions inline" and "show only comments and formatting in balloons" - when any of these three options is activated, the focus returns to the document and you can navigate and edit text as normal, however NVDA does not read anything while navigating.  Pressing NVDA+up reads the title ("Document! - Word").

Using the focus highlight addon, where normally the document itself is surrounded by a blue frame, when this happens there is a blue line around the whole Word program window (if not maximised, otherwise the line is just above the taskbar).

Pressing alt+tab to another program and back again fixes it.

**Comment by Adriani90** [comment]
@Qchristensen are you still able to reproduce this issue? Is it also occuring with UIA enabled?

**Comment by Qchristensen** [comment]
I can no longer reproduce this issue.  I can set the Balloons option to any of the three options, focus returns to the document and I can navigate and read as normal (the text at the caret is read while navigating, as expected).

I am using NVDA 2019.1.1 and Office 365, v16.0.11601.20174 Can anyone please confirm if this is also working in earlier versions of Office before we close this issue?

**Comment by Qchristensen** [comment]
Also just to note, this works whether the UIA setting in NVDA is checked or unchecked.

**Comment by Adriani90** [comment]
I tested with NVDA 2019.1.1 in Word 2010, I cannot reproduce this issue either. I am closing this as works for me. If someone still has this problem, please comment on it and we can reopen the discussion. Thanks.
